src/App.js

```diff
@@ -1,19 +1,29 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 import timelineData from './data/timeline.json';
 import Timeline from './components/Timeline';
 
+// Test Post
+// const events = [
+//   {
+//     person: 'Nica Cordeiro',
+//     status: 'Fabulous',
+//     timeStamp: '2016-04-13T11:11:03Z'
+//   }
+// ]
+
+
 function App() {
   console.log(timelineData);
 
   // Customize the code below
   return (
     <div className="App">
       <header className="App-header">
-        <h1 className="App-title">Application title</h1>
+        <h1 className="App-title">Ada Lovelace's Social Media Feed</h1>
```

Comment:
Just note that you can use the `timelineData` for the title.

```suggestion
        <h1 className="App-title">{timelineData.person}'s Social Media Feed</h1>
```